.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+- '0.12'
+- iojs
+sudo: false
+cache:
+  directories:
+  - node_modules
```

Comment:
I just copied this part from [webmaker-browser's `.travis.yml`](https://github.com/mozilla/webmaker-browser/blob/develop/.travis.yml).